snuba/web/views.py

```diff
@@ -187,7 +187,12 @@ def handle_invalid_dataset(exception: InvalidDatasetError) -> Response:
 def handle_invalid_query(exception: InvalidQueryException) -> Response:
     # TODO: Remove this logging as soon as the query validation code is
     # mature enough that we can trust it.
-    logger.warning("Invalid query", exc_info=True)
+    metrics.increment("query.invalid")
+    if exception.report:
+        logger.warning("Invalid query", exc_info=exception)
```

Comment:
Changed this to exception object since from Python 3.5, we can pass in exception object

From https://docs.python.org/3/library/logging.html#logging.Logger.debug
Changed in version 3.5: The exc_info parameter can now accept exception instances.

Reply:
Have you checked if anything changes in the event reported to sentry when we pass the exception itself ? If these gets rehashed and are grouped differently is fine, though I would verify that this approach still works for the sentry integration.